NOTICE.txt

```diff
@@ -0,0 +1,33 @@
+azure-sdk-for-c
+
+NOTICES AND INFORMATION
+Do Not Translate or Localize
+
+This software incorporates material from third parties. Microsoft makes certain
+open source code available at https://3rdpartysource.microsoft.com, or you may
+send a check or money order for US $5.00, including the product name, the open
+source component name, and version number, to:
+
+Source Code Compliance Team
+Microsoft Corporation
+One Microsoft Way
+Redmond, WA 98052
+USA
+
+Notwithstanding any other terms, you may reverse engineer this software to the
```

Comment:
missing space "Not withstanding..."

Reply:
This is verbatim text from our OSS Champ; I'm just going to keep it as is